new-integration-tests/src/test/java/oracle/weblogic/kubernetes/assertions/impl/Kubernetes.java

```diff
@@ -131,6 +131,26 @@ public static boolean isOperatorPodRunning(String namespace) throws ApiException
     return status;
   }
 
+  /**
+   * Checks if a Traefik pod running in a given namespace.
+   * The method assumes the traefik pod name to starts with "traefik-operator-"
+   * and decorated with label "app=traefik"
+   * @param namespace in which to check for the pod existence
+   * @return true if pod exists and running otherwise false
+   * @throws ApiException when there is error in querying the cluster
+   */
+  public static boolean isTraefikPodRunning(String namespace) throws ApiException {
```

Comment:
That the pod is RUNNING isn't very interesting.  We want to know that the pod is in the Ready condition.

Reply:
RUNNING just means that the pod has been assigned to a node.

Reply:
added a method to check the pod condition is ready